pyteal/ast/abi/array_static_test.py

```diff
@@ -255,7 +257,7 @@ def test_StaticArray_set_computed():
 
 @pytest.mark.parametrize("test_case", BYTES_SET_TESTCASES)
 def test_StaticBytes_set_py_bytes(test_case: bytes | bytearray):
-    value = abi.StaticBytes(len(test_case))
+    value: abi.StaticBytes[Literal[16]] = abi.StaticBytes(abi.StaticBytesTypeSpec(16))
```

Comment:
@fabrice102 any reason for this type def with explicit literal? Omitting `Literal[16]` should prevent mypy from complaining and we can get the expected length to pass with just `len(test_case)` so we don't need to hardcode 16

Reply:
My `mypy` was complaining without the typedef for some reasons.
I don't understand why though...

Reply:
Even with the `abi.StaticBytes` (instead of `abi.StaticBytes[Literal[16]]`)?

I just tried it locally and it complained with no type but was fine with omitted literal